internal/service/s3/bucket_website_configuration.go

```diff
@@ -142,6 +147,17 @@ func ResourceBucketWebsiteConfiguration() *schema.Resource {
 					},
 				},
 			},
+			"routing_rules": {
```

Comment:
currently not entirely sure if we should keep both params or deprecate `routing_rule` in favor of this one 🤔

Reply:
hmm or we can always deprecate one of these later if for example we migrate to the plugin-framework and that better supports empty string values in nested types